micrometer-test/src/test/java11/io/micrometer/core/instrument/binder/jdk/JdkHttpClientTimingInstrumentationVerificationTests.java

```diff
@@ -13,10 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.micrometer.core.instrument;
+package io.micrometer.core.instrument.binder.jdk;
```

Comment:
The `HttpClientObservationDocumentation` is package private, which is why I moved the test here. Maybe it's better to make it public instead? Though I'm not sure why someone outside Micrometer would need to access it.